docs/user-guide/src/bmo/root_device_hints.md

```diff
@@ -0,0 +1,119 @@
+device hints][ironic-hints], but there are important differences in available
+hints and the comparison operators they use.
+
+**Warning:** the default root device depends on the hardware profile as
+explained below. Currently, `/dev/sda` path is used when no hints are
+specified. This value is not going to work for NVMe storage. Furthermore, Linux
+does not guarantee the block device names to be consistent across reboots.
+
+[ironic-hints]: https://docs.openstack.org/ironic/latest/install/advanced.html#specifying-the-disk-for-deployment-root-device-hints
+
+## RootDeviceHints format
+
+One or more hints can be provided, the chosen device will need to match all of
+them. Available hints are:
+
+- ``deviceName`` -- A string containing a canonical Linux device path like
+  `/dev/vda` or a *by-path* alias like `/dev/disk/by-path/pci-0000:04:00.0`.
+
+  **Warning:** as mentioned above, block device names are not guaranteed to be
+  consistent across reboots. If possible, choose a more reliable hint, such as
+  `wwn` or `serialNumber`.
+
+  **Hint:** only *by-path* aliases are supported, other aliases, such as
+  *by-id* or *by-uuid*, cannot currently be used.
+
+- `hctl` -- A string containing a SCSI bus address like `0:0:0:0`.
+
+- `model` -- A string containing a vendor-specific device
+  identifier. The hint can be a substring of the actual value.
+
+- `vendor` -- A string containing the name of the vendor or
+  manufacturer of the device. The hint can be a substring of the
+  actual value.
+
+- `serialNumber` -- A string containing the device serial number.
+
+- `minSizeGigabytes` -- An integer representing the minimum size of the
+  device in Gigabytes.
+
+- `wwn` -- A string containing the unique storage identifier.
+
+- `wwnWithExtension` -- A string containing the unique storage
+  identifier with the vendor extension appended.
+
+- `wwnVendorExtension` -- A string containing the unique vendor
+  storage indentifier.
+
+- `rotational` -- A boolean indicating whether the device must be
+  a rotating disk (`true`) or not (`false`). Examples of non-rotational devices
+  include SSD and NVMe storage.
+
+## Finding the right hint value
+
+Since the root device hints are only required for provisioning, you can use the
+results of inspection to get an overview of available storage devices:
+
+```bash
+kubectl get hardwaredata/<BMHNAME> -n <NAMESPACE> -o jsonpath='{.spec.hardware.storage}' | jq .
+```
+
+On a virtual testing environment, this commands outputs
+
+```json
+[
+  {
+    "alternateNames": [
+      "/dev/sda",
+      "/dev/disk/by-path/pci-0000:03:00.0-scsi-0:0:0:0"
+    ],
+    "hctl": "0:0:0:0",
+    "model": "QEMU HARDDISK",
+    "name": "/dev/disk/by-path/pci-0000:03:00.0-scsi-0:0:0:0",
+    "rotational": true,
+    "serialNumber": "drive-scsi0-0-0-0",
+    "sizeBytes": 32212254720,
+    "type": "HDD",
+    "vendor": "QEMU"
+  }
+]
+```
+
+Here you can find all necessary fields to populate the root device hints before
```

Comment:
```suggestion
Here you can find all necessary fields to populate the root device hints before
```
Here is typically used to provide a link or following output, but now its after the output, so here is weird to me.  Could this string be placed above the output?

Reply:
Let me see